src/game/savegame/savegame.c

```diff
@@ -831,6 +831,7 @@ ReadGame(const char *filename)
 		{"YQ2-3", 3},
 		{"YQ2-4", 4},
 		{"YQ2-5", 5},
+		{"YQ2-6", 6},
```

Comment:
Is adding the `music` field to `spawn_temp_t` backward compatible or does this mess with the overall state when loading a v5 savegame into a v6 game library?

Reply:
I have not found any issues with load 'music' in code without such commit and with it. It was mostly safety check, I have removed this change for now. I suppose we could create such check in future if someone have found some issue.